ui/src/app/base/components/FormCardButtons/FormCardButtons.js

```diff
@@ -20,6 +24,18 @@ export const FormCardButtons = ({
       >
         Cancel
       </Button>
+      {handleSecondarySubmit && secondarySubmitLabel && (
+        <Button
+          appearance="neutral"
+          className="u-no-margin--bottom"
+          data-test="secondary-submit"
```

Comment:
@huwshimi I think they're okay in production code, they affect performance or alter behaviour. In fact I think we probably *want* them in the production build for e2e tests.

Reply:
agreed with @squidsoup that it'll be useful for e2e tests, but it's actually already useful because it makes these unit tests less brittle. Usually their purpose isn't to test that the DOM has a certain shape, so why bind a test to the DOM shape when all you care about is a single button?